elliott/elliottlib/errata.py

```diff
@@ -770,3 +770,24 @@ def put_file_meta(advisory_id, file_meta: dict) -> List[dict]:
     """
     return ErrataConnector()._put(f'/api/v1/erratum/{advisory_id}/filemeta?put_rank=true',
                                   json=file_meta)
+
+
+def push_cdn_stage(advisory_id):
+    """Trigger stage push for an advisory
+    https://errata.devel.redhat.com/documentation/developer-guide/api-http-api.html#pushing-advisories
+    """
+    connector = ErrataConnector()
+    push_log = connector._get(f'/api/v1/erratum/{advisory_id}/push')
```

Comment:
This can reside in it's own func `get_completed_pushes`, upto you if you wanna do that

Reply:
I can't find such a function, where did you find it? I can only get push() function in https://github.com/red-hat-storage/errata-tool/blob/master/errata_tool/erratum.py#L993

Reply:
lol, erratum lib has push function we can use it